packages/r/rax.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "rax",
+  "filename": "rax.min.js",
+  "description": "A universal React-compatible render engine.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/alibaba/rax"
+  },
+  "keywords": [
+    "rax",
+    "react",
+    "jsx"
+  ],
+  "homepage": "https://rax.js.org/",
+  "npmName": "rax",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
+      ]
+    }
+  ],
```

```suggestion
  "autoupdate": {
    "source": "npm",
    "target": "rax",
    "fileMap": [
      {
        "basePath": "dist",
        "files": [
          "**/*"
        ]
      }
    }
  ],
```
Please use the new `autoupdate` format documented in CONTRIBUTING.md :)

---

Fixed.